Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.10-slim
+
+ARG api_port
+ENV port ${api_port}
+
+ARG root_path
+ENV path ${root_path}
+
+WORKDIR /root
+COPY ./requirements.txt /root/requirements.txt
+
+# required to install github.com/TheRealNoob/python-logging-loki.git
+RUN apt update && \
+    apt install git curl -y && \
+    apt clean autoclean && \
+    apt autoremove --yes && \
+    rm -rf /var/lib/{apt,dpkg,cache,log}
+
+RUN pip install --no-cache-dir -r /root/requirements.txt
+
+COPY . /root/
+
+CMD ["uvicorn", "api.app:app", "--proxy-headers", "--host 0.0.0.0", "--port ${port}", "--root-path", "${path}"]
```

Comment:
i think it should be
`CMD ["uvicorn", "api.app:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "${port}", "--root-path", "${path}"]`
like it was in the past
`CMD ["uvicorn", "api.app:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "5000", "--root-path", "/dev"]`

Reply:
I'm running into errors with the suggested format also. The env vars doesn't seem to be going in correctly..

This works: `CMD uvicorn api.app:app --proxy-headers --host 0.0.0.0 --port ${port} --root-path ${path}` but I don't think that is the syntax we want to use.

Reply:
`Error: Invalid value for '--port': ' $port' is not a valid integer.`

Reply:
Same issue with ${port} syntax as well.

Reply:
@extreme4all @TheRealNoob I can't get anything except for `CMD uvicorn api.app:app --proxy-headers --host 0.0.0.0 --port ${port} --root-path ${path}` to work. https://github.com/moby/moby/issues/5509 is a good synopsis of why this is the case.